### ISSUE TITLE
Support CLI option `exit` to force exit after mocha done running the tests

### ISSUE THREAD

**Issue by sairamnutheti** [krakenjs/nemo#29]
In mocha versions <4, mocha kill itself after it is done with running all the tests. From mocha versions >=4 mocha will be still alive when any active handles are still on the event loop and the process will just hang. If we are running mocha through command line there is an option to handle it by passing `--exit` option to mocha. Since we are using mocha programmatically in Nemo, we cannot pass it as a mocha option in the config. So, it would be great if we have a CLI option that we can pass to Nemo something like this `nemo --exit` to force exit by shutting down the event loop. We can keep the default behavior of `no-exit` if it is not passed as command line argument

**Comment by grawk** [comment]
Can you please provide a reproducible case with nemo to illustrate the behavior?

**Comment by sairamnutheti** [comment]
```const net = require('net');

describe('@no-exit@', function () {

  before(function () {
    const server = net.createServer();
    server.listen(8000, function () {
      console.log('listening on port: 8000');
    });
  });

  it('Should launch a browser with given url', async function () {
    const {nemo} =  this;
    await nemo.driver.get('https://www.paypal.com');
  });
});```

**Comment by grawk** [comment]
Ah. Thanks. That's clarifying..

**Comment by grawk** [comment]
Fixed in `v4.6.0`
